samples/client/petstore/typescript-angular/api.module.ts

```diff
@@ -0,0 +1,9 @@
+import * as api from './api/api';
```

Comment:
I used this as the angular2 client was doing the similar thing such as https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache#L10

Open to suggestion to change or leave as is.

An explicit import could be made. Out of interest, what were the problems you ran into?